WPMedia/fusion-news-theme-blocks
# New custom fields for Nav Landmark HTML Elements

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description
This PR addresses the `nav` elements that are used as landmarks for screen readers. I included new custom fields for these blocks so that clients may customize what that text is, in case they use those blocks in multiple places throughout their sites.

## Jira Ticket
- [TMEDIA-55](https://arcpublishing.atlassian.net/browse/TMEDIA-55)

## Acceptance Criteria
If a `nav` landmark is used that is not the primary navigation landmark of the page should have an identifiable label
Using the `aria-label` attribute on the `nav` element to add a label value.

## Test Steps

1. Checkout this branch: `git checkout TMEDIA-55/nav_landmark_identifier`
2. Run fusion repo: `npx fusion start -f -l @wpmedia/alert-bar-block,@wpmedia/header-nav-chain-block,@wpmedia/links-bar-block`
3. Go to http://localhost/pagebuilder/editor/curate/?p=p0ciQAinp2VIRr&v=vFzdS81np2VIRr
4. Under `Features`, observe in `Header Nav Chain`, `Alert Bar`, and `Links Bar`
5. Observe the new Custom field for Aria Labels.
6. On http://localhost/pf/all-blocks/?_website=the-sun, use a landmark identifier(for example: http://matatk.agrip.org.uk/landmarks/), observe the `Navigation` elements that are marked the same as the aria-label in their corresponding HTML node in the inspector. These are default values.
7. You can update the Aria label fields for these blocks, Share & Publish, and observe the change in the inspector and the landmark identifier

## Effect Of Changes
### Before

All landmarks were labeled `Navigation`

### After


Landmarks are labeled according to custom field with default values.
![Screen Shot 2021-05-14 at 3 55 01 PM](https://user-images.githubusercontent.com/82830431/118330132-f31ca180-b4cc-11eb-862a-69bc987c231b.png)
![Screen Shot 2021-05-14 at 3 58 21 PM](https://user-images.githubusercontent.com/82830431/118330330-3c6cf100-b4cd-11eb-80e8-ecedc14250e2.png)


## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
